api/app/geotiff_from_stac_api.py

```diff
@@ -0,0 +1,84 @@
+import os
+from uuid import uuid4
+
+import boto3
+from cachetools import TTLCache, cached
+from fastapi import HTTPException
+from odc.geo.xr import write_cog
+from odc.stac import configure_rio, stac_load
+from pystac_client import Client
+
+STAC_URL = "https://api.earthobservation.vam.wfp.org/stac"
```

Comment:
How about making this a bit more future-proof and accepting the STAC_URL as a parameter?

Not urgent but this is something that will be needed if we want to download data from other sources.

Reply:
The issue is that it is not only a URL but also an authenticated S3 bucket